### DISCUSSION TITLE
Buttons for start and stop all games

### DISCUSSION THREAD

**Discussion by Blarzek** [AlexanderSharykin/CardIdleRemastered#47] [Ideas]
Hello, thank you so much for your work.

As the title says, could you add buttons that allow us to start all games at once, and to stop them all too?
It would be very useful.

Thank you for your attention.
Have a nice day!

**Comment by AlexanderSharykin** [comment]
Hello, thank you :)

Buttons that "allow to start all games at once, and to stop them all too" are on Queue tab

To run multiple games at once, add them to the Queue and select "All together" option for Queue

You can add games to the Queue from Badges tab, either one by one or all at once via buttons at the bottom of the table

Steam doesn't recognize more than 30 running games, so that will be the limit if you have more games with cards. You can also set _smaller_ number in Settings (but 30 is fine)

Hope that is sufficient for your purposes, enjoy

**Comment by Blarzek** [comment]
Ohh, I see. Thank you so much for your response!
